### PR TITLE
Add Micro Average Accuracy

### PR DESCRIPTION
**Pull request by wistuba**
By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

### CODE REVIEW

File: src/renate/benchmark/experimentation.py [hunk 1]
```diff
@@ -72,12 +75,13 @@ def cumulative_metrics_summary(
             metrics.
         cumulative_metrics: The list of (name, metric) tuples.
         num_tasks: The total number of tasks.
+        num_instances: Count of test data points for each task.
     """
     data = []
-    for task_id in range(num_tasks + 1):
+    for task_id in range(num_tasks):
```
Comment (lballes) [inline src/renate/benchmark/experimentation.py:81]: Why did this change? I don't understand why it was `num_tasks + 1` before, but this change doesn't seem related to the micro-average accuracy.

Comment (wistuba) [reply src/renate/benchmark/experimentation.py:81]: Agreed, not related to micro average at all, just some clean up.

In L384 the `-1` was removed so that would be the adequate change in this function.

**Review by lballes — APPROVED** [review]
